src/cheri_insts.sail

```diff
@@ -229,6 +230,46 @@ function clause execute (CGetOffset(rd, cs1)) = {
   RETIRE_SUCCESS
 }
 
+/*!
+ * Integer register *rd* is set equal to the **upper half** of capability
+ * register *cs*.
+ *
+ * The bits returned here are of the **in-memory** form of the capability, which
+ * differs from the compressed in-register form by the XOR of [null_cap_bits].
+ * That is, applying [CGetHigh] to a capability loaded from address *m* will
+ * yield the same result as loading the upper half of the capability-sized
```

Comment:
Is `upper` suffiently clear for a hypothetical big-endian world? Maybe add a qualification such as `(i.e. bits XLEN..2*XLEN-1 when interpreted as a 2*XLEN integer)`?